docs/README.md

```diff
@@ -0,0 +1,5 @@
+## Quarkus Documentation
+
+This is the main folder for Qaurkus documentation. 
+
+This folder is synchronized to website and the [Primary Documentatin README.adoc](src/main/asciidoc/README.adoc) are inside the `src/main/asciidoc` folder.
```

Comment:
```suggestion
This folder is synchronized to website and the [Primary Documentation README.adoc](src/main/asciidoc/README.adoc) are inside the `src/main/asciidoc` folder.
```

Reply:
Suggesting and alternative phrasing. I leave it up to you to accept or decline this change.
```suggestion
The contents of this folder are used to publish documentation to the quarkus.io/docs website.
The [main documentation README.adoc](src/main/asciidoc/README.adoc) is in the `src/main/asciidoc` folder that also contains the `.adoc` source files for the published documentation.
```